optuna/samplers/_nsga2/sampler.py

```diff
@@ -87,13 +152,18 @@ class NSGAIISampler(BaseSampler):
                 versions without prior notice. See
                 https://github.com/optuna/optuna/releases/tag/v2.5.0.
 
+    Raises:
+        ValueError:
+            If ``crossover`` is `undx` or `spx` and `population_size` is less than or equal to two
+            (To use three parental individuals during crossover applied).
```

Comment:
I think this should be documented as follows.
- To be raised `ValueError` if ``crossover`` is not in `[uniform, blxalpha, sbx, vsbx, undx, spx]`.
- To be raised `ValueError` if ``population_size`` is smaller than ``n_parents``. The ``n_parents`` is determined by each crossover algorithm like ~~~.

Reply:
The documentation has been fixed.